packages/files/src/main.rs

```diff
@@ -93,8 +125,31 @@ fn transform_image(input: Value, to: &str) {
             match fs::read(path) {
                 Ok(contents) => {
                     let encoded: String = general_purpose::STANDARD_NO_PAD.encode(contents);
-                    let html = format!("<img src=\"data:image/png;base64, {encoded} \"/>");
-                    let json = json!({"name": "raw", "data": html}).to_string();
+                    let width = input["arguments"]["width"].as_f64().unwrap();
+                    let caption = input["arguments"]["caption"].as_str().unwrap();
```

Comment:
I think we should support ModMark markup inside of the caption, so perhaps wrap the caption inside of a `inline_content` module?  (Or alternativly a `__text` module. With the current behaviour we allow users to inject raw content directly, see screenshot.  

![image](https://user-images.githubusercontent.com/11508459/230428913-675fa199-dfc7-4016-b9ad-91efccd094a9.png)